hcloud/images/client.py

```diff
@@ -22,22 +22,59 @@ def __init__(self, client, data):
 
     def get_actions_list(self, sort=None, page=None, per_page=None):
         # type: (Optional[List[str]], Optional[int], Optional[int]) -> PageResult[BoundAction, Meta]
+        """Returns a list of action objects for the image.
+
+        :param sort: List[str] (optional)
+               Specify how the results are sorted. See `our documentation <https://docs.hetzner.cloud/#actions-list-all-actions>`_  for all available values.
```

Comment:
Put all possible sort choices here and refer to API docs only for details. Or document all possible sort choices on the class level (or on module level).

Reply:
What if the strings would be changed in the future? We would need to release a new hcloud-python version then, instead of just using our API Documentation which should be the single point of truth.